src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -246,6 +246,11 @@ def load_command_table(self, _):
         g.custom_command('update', 'cli_cosmosdb_table_throughput_update')
         g.custom_command('migrate', 'cli_cosmosdb_table_throughput_migrate')
 
+    with self.command_group('cosmosdb identity', None, client_factory=cf_db_accounts) as g:
```

Comment:
you could add `is_preview=True` in command group level instead of several preview tag in each command.

Reply:
`None` is strange here.  If you don't want to declare this parameter, just skip it.